.devcontainer/claude-code-copilot-api/devcontainer.json

```diff
@@ -43,9 +43,9 @@
 	// Use 'postCreateCommand' to run commands after the container is created.
 	"postCreateCommand": {
 		"disable metrics": "skaffold config set --global collect-metrics false",
-		"make toolchain": "make toolchain",
 		"configure pre-commit": "pre-commit install --install-hooks",
-		"install copilot-api": "yarn global add tallaxes/copilot-api#npm-github" // temp from fork until upstream is fixed
+		"install copilot-api": "yarn global add tallaxes/copilot-api#npm-github", // temp from fork until upstream is fixed
+		"make toolchain": "make toolchain"
```

Comment:
No functional changes, just order (even though these are all run in parallel, codespace only shows the last one as "in progress", and it is actually `make toolchain` that takes time)